src/app.css

```
@@ -141,8 +146,7 @@
     .text-default-m-nc {
         @apply text-default-nc font-medium;
     }
-
-    .highlight-on-hover {
+    .text- .highlight-on-hover {
```

Same here, `text-h3` isn't used.
```suggestion
    .highlight-on-hover {
```

---

I fixed this, but highlight-on-hover is used.